### DISCUSSION TITLE
Make Ctrl+Scroll adjust beat snap divisor by default

### DISCUSSION THREAD

**Discussion by kongehund** [ppy/osu#31060] [Ideas] [answered]
This was mentioned in https://github.com/ppy/osu/discussions/28669 but I think a separate thread makes sense.

**In stable:**
- Ctrl+Scroll adjusts beat snap divisor
- Alt+Scroll adjusts either Distance Spacing or Volume, depending on cursor position

**In lazer:**
- Ctrl+Scroll adjusts volume
- Alt+Scroll ALSO adjusts volume
- Ctrl+Shift+Scroll adjusts beat snap divisor

It is unnecessary to have two modifier keys control volume when scrolling. Stable mappers do not have muscle memory associating Ctrl with volume control, only Alt. Therefore, Ctrl can be freed up to match the muscle memory from stable, such that Ctrl+Scroll adjusts the beat snap divisor, in addition to Ctrl+Shift+Scroll

Although Ctrl+Shift+Scroll exists and although the binding can be changed in the options, I see little reason not to have Ctrl+Scroll control beat snap divisor by default.

**Comment by peppy** [comment]
FWIW, Ctrl+Alt can be used for distance spacing adjustment. Also both of these are customisable in settings.

But I can agree with fixing the default for beat snap divisor since it's not being used by anything else.

**Reply by peppy** [reply]
Of course it's not easy to make this happen. Generally it seems that we don't want the concept of fall-through volume adjustment to exist at a game level. If anything, it should be local to screens, or implemented at `OsuScreen` as a `virtual bool` so screens can choose if they want this functionality in the first place.

**Comment by kongehund** [comment, marked as answer]
Applied in https://github.com/ppy/osu/commit/d97ea781364323383fa59512e45cac494387fb4b , thanks peppy
